src/components/PatientLookup.tsx

```diff
@@ -68,7 +70,9 @@ const PatientLookup: React.FC<PatientLookupProps> = ({ isOnDashboard }) => {
 
   return (
     <React.Fragment>
-      <div className="InputRow">
+      <Stack>
+        <StackItem className="input-row-label">Create a new Predictive analysis</StackItem>
```

Comment:
Should this be "Create a new predictive analysis" or "Create a New Predictive Analysis"?

The capitalization seems a bit inconsistent.

@colinywu @janakitti what are your thoughts on this?

Reply:
Resolving this comment, has been fixed in another PR.